### PR TITLE
Fix LocationHierarchy.getLocationId fails for Pagination endpoints 🐛

### PR DESCRIPTION
**Pull request by ndegwamartin**
- Fixes LocationHierarchy.getLocationId fails for Pagination endpoints 
- Release version 2.2.5

**Engineer Checklist**

- [x] I have written **Unit tests** for any new feature(s) and edge cases for
      bug fixes
- [x] I have added documentation for any new feature(s) and configuration
      option(s) on the `README.md`
- [x] I have run `mvn spotless:check` to check my code follows the project's
      style guide
- [x] I have run `mvn clean test jacoco:report` to confirm the coverage report
      was generated at `plugins/target/site/jacoco/index.html`
- [x] I ran `mvn clean package` right before creating this pull request.

### CODE REVIEW

File: plugins/src/test/java/org/smartregister/fhir/gateway/plugins/UtilsTest.java [hunk 1]
```diff
@@ -10,13 +10,7 @@
...
 
 import org.hl7.fhir.instance.model.api.IBaseBundle;
-import org.hl7.fhir.r4.model.Base64BinaryType;
-import org.hl7.fhir.r4.model.Binary;
-import org.hl7.fhir.r4.model.Bundle;
-import org.hl7.fhir.r4.model.Composition;
-import org.hl7.fhir.r4.model.Identifier;
-import org.hl7.fhir.r4.model.Meta;
-import org.hl7.fhir.r4.model.Reference;
+import org.hl7.fhir.r4.model.*;
```
Comment (Rkareko) [inline plugins/src/test/java/org/smartregister/fhir/gateway/plugins/UtilsTest.java:13]: Is the use of wildcard import intentional?

File: plugins/src/main/java/org/smartregister/fhir/gateway/plugins/LocationHierarchyEndpointHelper.java [hunk 2]
```diff
@@ -14,13 +14,7 @@
...
 
 import org.hl7.fhir.instance.model.api.IBaseBundle;
-import org.hl7.fhir.r4.model.Binary;
-import org.hl7.fhir.r4.model.Bundle;
-import org.hl7.fhir.r4.model.Composition;
-import org.hl7.fhir.r4.model.ListResource;
-import org.hl7.fhir.r4.model.Location;
-import org.hl7.fhir.r4.model.Resource;
-import org.hl7.fhir.r4.model.StringType;
+import org.hl7.fhir.r4.model.*;
```
Comment (Rkareko) [inline plugins/src/main/java/org/smartregister/fhir/gateway/plugins/LocationHierarchyEndpointHelper.java:17]: Is the use of wildcard import intentional?

Comment (ndegwamartin) [reply plugins/src/main/java/org/smartregister/fhir/gateway/plugins/LocationHierarchyEndpointHelper.java:17]: Same as above

**Review by Rkareko — APPROVED** [review]
